base/system/diskpart/diskpart_msg.mc

```diff
@@ -2037,105 +2043,110 @@ Example:
-                    create EFI System partitions, the CREATE PARTITION MSR
-                    command to create Microsoft Reserved partitions, and the
-                    CREATE PARTITION PRIMARY command without this parameter to
-                    create primary partitions on GPT disks.
-
-    ALIGN=<N>   Typically used with hardware RAID Logical Unit Number (LUN)
-                arrays to improve performance. The partition offset will be
-                a multiple of <N>. If the OFFSET parameter is specified, it
-                will be rounded to the closest multiple of <N>.
-
-    NOERR       For scripting only. When an error is encountered, DiskPart
-                continues to process commands as if the error did not occur.
-                Without the NOERR parameter, an error causes DiskPart to exit
-                with an error code.
-
-    After you create the partition, the focus automatically shifts to the new
-    partition. The partition does not receive a drive letter. You must use the
-    assign command to assign a drive letter to the partition.
-
-    A basic disk must be selected for this operation to succeed.
-
-    If a partition type is not specified, the disk is uninitialized and disk
-    size is greater than 2TB, it will be initialized to GPT.
-
-Example:
+                Dacă acest parametru nu este specificat pentru un disc GPT,
+                comanda crează o partiție de bază pentru date.
+
+                Orice octet care descrie tipul de partiție sau GUID poate fi
+                specificată cu acest parametru. DiskPart nu verifică tipul
+                partiției pentru validitate, cu excepția faptului că este
+                un octet cu valoare în format hexazecimal sau un GUID.
+
+                Atenție:
+
+                    Creând partiții cu acest parametru poate cauza
+                    calculatorului dumneavoastră să nu mai funcționeze bine sau
+                    să nu mai poată porni. Dacă nu sunteți un OEM sau un
+                    profesionist cu experiență în discuri GPT, nu creați
+                    partiții pe discuri GPT folosind acest parametru.
+                    În schimb, folosiți întotdeauna comanda CREATE PARTITION EFI ca să creați
+                    partiții de sistem EFI, comanda CREATE PARTITION MSR ca să
+                    creați partiții rezervate Microsoft, și comanda
+                    CREATE PARTITION PRIMARY fără acest parametru ca să creați
+                    partiții primare pe discuri GPT.
+
+    ALIGN=<N>   Folosită deobicei cu Numărul unității logice RAID (LUN)
+                a șirurilor de dispozitive pentru a îmbunătăți performanța.
+                Decalajul partiției va fi un multiplu a lui <N>. Dacă
+                parametrul OFFSET este specificat, va fi rotunjit la cel mai
+                apropiat multiplu a lui <N>.
+                                                
+    NOERR       Doar pentru automatizare. Când apare o eroare, DiskPart
```

Comment:
"For scripting only" gets translated here as "doar pentru automatizare", but in other instances as "doar pentru scriere de fișiere script". Please take care of consistency.